posts/parent-class/index.qmd

```diff
@@ -0,0 +1,50 @@
+---
+title: "Choosing the Right Parent for R Object Classes"
+author:
+  - name: "Hugo Gruson"
+    orcid: "0000-0002-4094-1476"
+date: "2024-06-24"
+categories: [R, R package, object-oriented programming, S3, interoperability]
+---
+
+[I have recently published](https://hugogruson.fr/posts/compa-tibble/) a [series of blog posts](https://epiverse-trace.github.io/posts/s3-generic/) on the reasons why one may want to start using object-oriented programming (and more specifically R S3 classes) to improve interoperability with other tools from the ecosystem.
+
+But there are still questions I have not addressed directly, even if they may have been implicitly included sometimes: what makes a good object class? What good practices in class & function design can improve interoperability?
+
+As you can expect from these questions, this post may be slightly more subjective than the previous ones on this topic.
```

Comment:
I think you can rather straightforwardly state that the post is to provide a rather subjective opinion on the matter.